lib/lib-rds/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@aws-sdk/lib-rds",
+  "version": "3.43.0",
+  "description": "RDS utility for generating a password that can be used for IAM authentication to an RDS DB.",
+  "main": "./dist-cjs/index.js",
+  "module": "./dist-es/index.js",
+  "types": "./dist-types/index.d.ts",
+  "scripts": {
+    "build": "yarn build:cjs && yarn build:es && yarn build:types",
+    "build:cjs": "tsc -p tsconfig.cjs.json",
+    "build:es": "tsc -p tsconfig.es.json",
+    "build:types": "tsc -p tsconfig.types.json",
+    "downlevel-dts": "downlevel-dts dist-types dist-types/ts3.4",
+    "test": "jest"
+  },
+  "engines": {
+    "node": ">= 10.0.0"
+  },
+  "author": {
+    "name": "AWS SDK for JavaScript Team",
+    "url": "https://aws.amazon.com/javascript/"
+  },
+  "license": "Apache-2.0",
+  "dependencies": {
+    "@aws-sdk/signature-v4": "3.40.0",
+    "@aws-sdk/protocol-http": "3.40.0",
```

Comment:
Some dependencies gone missing here: like `@aws-sdk/credential-provider-node`, `@aws-crypto/sha256-js`. This is causing the CI build to fail.